src/main/java/de/tum/cit/aet/artemis/atlas/service/competency/CompetencyProgressService.java

```diff
@@ -129,19 +126,6 @@ public void updateProgressByCompetencyAsync(CourseCompetency competency) {
         existingProgress.stream().map(CompetencyProgress::getUser).forEach(user -> updateCompetencyProgress(competency.getId(), user));
     }
 
-    /**
-     * Asynchronously update the progress of all users in the course for a specific competency
-     *
-     * @param competency The competency for which to update all existing student progress
-     */
-    @Async
-    public void updateProgressByCompetencyAndUsersInCourseAsync(CourseCompetency competency) {
-        SecurityUtils.setAuthorizationObject(); // Required for async
-        Set<User> users = userRepository.getUsersInCourse(competency.getCourse());
-        log.debug("Updating competency progress for {} users.", users.size());
-        users.forEach(user -> updateCompetencyProgress(competency.getId(), user));
-    }
```

Comment:
Why is this not needed anymore?

Reply:
The method was unused even before this PR, so it is save to delete